assets/src/block-editor/components/sidebar-notification/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import { Loading } from '../../../components/loading';
+
+/**
+ * Notification component used in the block editor sidebar.
+ *
+ * @param {Object} props
+ * @param {Object} props.action Call to action element.
+ * @param {Object} props.icon Status icon element.
```

Comment:
Good catch.

It actually seems that using a `ReactChild` might be even better since both we accept either elements or strings here:

```js
type ReactText = string | number;
type ReactChild = ReactElement | ReactText;
```

What do you think, @pierlon?

Reply:
@pierlon I've addressed this one in efeba44. I've used the `ReactChild` but feel free to amend if you think another type is better-suited.